### PR TITLE
More robust death reporting

### LINKED ISSUE #23413: More robust death reporting
Right now, deaths are reported with a lot of really useful information, but they're missing two very important pieces of data.

1. The server the death occurred on  
and  
2. The map file the death occurred on

Right now, the only way I can think of to determine which map a death occurred on would be to compare the coordinates against an array of areas, but that's a pain in the butt.

Ideally, the server/map info would go in a new column on the death table. Seeing as how SQL schema changes aren't easy to do, simply appending that data to the `ss13death.coord` column would work. Format's not super important, but if I got my wish, it'd look like:  
`(Xx,Yy,Zz)-MapName-ServerAddr`

EDIT: For the sake of backwards compatibility, the format should be:  
`Xx,Yy,Zz,MapName,ServerAddr`

### PR DESCRIPTION
**Pull request by Cyberboss**
Closes #23413 

@nfreader

### CODE REVIEW

File: code/orphaned_procs/statistics.dm [hunk 1]
```diff
@@ -63,7 +63,8 @@
 		laname = sanitizeSQL(H.lastattacker:real_name)
 		lakey = sanitizeSQL(H.lastattacker:key)
 	var/sqltime = time2text(world.realtime, "YYYY-MM-DD hh:mm:ss")
-	var/coord = "[H.x], [H.y], [H.z]"
+	var/url = "byond://[world.address]:[world.port]"
```
Comment (lzimann) [inline code/orphaned_procs/statistics.dm]: You are adding the server url but not actually doing anything with it...

File: code/orphaned_procs/statistics.dm [hunk 2]
```diff
@@ -63,7 +63,8 @@
...
+	var/coord = "[H.x], [H.y], [H.z], [MAP_NAME], [url]"
```
Comment (Cyberboss) [inline code/orphaned_procs/statistics.dm]: @lzimann

Comment (lzimann) [inline code/orphaned_procs/statistics.dm]: oh i'm blind

Comment (lzimann) [inline code/orphaned_procs/statistics.dm]: btw there's also a report_cyborg_death in the same file, might want to change that aswell

**Comment by MrStonedOne** [comment]
This breaks database normalization rules.

Make a new column

**Comment by Jordie0608** [comment]
Ideally one for both new values.

**Comment by Cyberboss** [comment]
sql not my forte

**Pull request closed without merging**